engine/crates/engine-v2/id-derives/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
```

Comment:
Don't think I would have created a new crate for it, but 🤷

Reply:
I don't think you would have had a choice, proc macros live in proc macro crates.

Reply:
wow ok interesting, thanks for pointing it out

Reply:
It's because they have to be compiled into a shared library and then the rust compiler does some dynamic linking at runtime to actually pull the code in.  

Way simpler to have them isolated in their own crates than have users (and the compiler) deal with the burden of figuring out which bits are macro-time code and which bits are runtime code.